src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -198,4 +223,20 @@ public static Priority parsePriority(String priorityStr) throws ParseException {
         }
         return new Priority(trimmedPriority);
     }
+
+    /**
+     * Parses {@code Collection<String> appointments} into a {@code Set<Appointment>}.
+     */
+    public static Set<Appointment> parseAppointments(Collection<String> appointments) throws ParseException {
```

Comment:
Just a nit: Could have a better naming for parseAppointments as parseAppointment exists, this might cause some bugs due to naming. Maybe parseMultipleAppointment?

Reply:
Right! I think this is mainly for manual parsing / json text parsing, so I'll see how I can rename it (since appt takes multiple strings to construct)